### ISSUE TITLE
A suggestion for teleport function

### ISSUE THREAD

**Issue by DimkaAkuma** [modmuss50/SimpleVoidWorld#9]
It would be nice if spawn coordinates in void dimension  would always be the same(or configurable), independent from Teleport Block coordinates in Overworld, thus resulting in possible clay harvest abuse removal possibility, also you wouldn't need to worry, if you decided to move a block  in overworld to other position.

**Comment by modmuss50** [comment]
I'm thinking of making it impossible to break the clay blocks. That will be the easiest solution.

**Comment by modmuss50** [comment]
Or have them not drop anything when broken.

**Comment by DimkaAkuma** [comment]
But you stay then glued to one position in overworld - kinda sad

**Comment by orionSquared** [comment]
I agree with this suggestion. If you could make the coords in the overworld match the coords in the void world, it'll allow multiple people to enter the void dimension to create bases that are away from each other (assuming they're away from each other in the real world). This also allows you to remove the functionality that right clicking to the overworld takes you to a bed, and you can just right click to take you back to the void portal you placed down.
